cvmfs/swissknife_check.cc

```diff
@@ -128,6 +128,16 @@ bool CommandCheck::CompareEntries(const catalog::DirectoryEntry &a,
              a.name().c_str(), b.name().c_str());
     retval = false;
   }
+  if (diffs & Difference::kUid) {
+    LogCvmfs(kLogCvmfs, kLogStderr, "uids differ: %d / %d (%s / %s)",
+             a.uid(), b.uid(), a.name().c_str(), b.name().c_str());
+    retval = false;
+  }
+  if (diffs & Difference::kGid) {
+    LogCvmfs(kLogCvmfs, kLogStderr, "gids differ: %d / %d (%s / %s)",
+             a.gid(), b.gid(), a.name().c_str(), b.name().c_str());
+    retval = false;
+  }
```

Comment:
```suggestion
  if (!is_transition_point) {
      if (diffs & Difference::kUid) {
        LogCvmfs(kLogCvmfs, kLogStderr, "uids differ: %d / %d (%s / %s)",
                 a.uid(), b.uid(), a.name().c_str(), b.name().c_str());
        retval = false;
      }
      if (diffs & Difference::kGid) {
        LogCvmfs(kLogCvmfs, kLogStderr, "gids differ: %d / %d (%s / %s)",
                 a.gid(), b.gid(), a.name().c_str(), b.name().c_str());
        retval = false;
      }
    }
```
This check should not should fail for nested catalog transition points. There could be a `catalog-chown` that is not expected to affect them, I think.

Reply:
Commiting in order to start cloudtests